### ISSUE TITLE
Жрёт невменяемую нагрузку на запуске браузера.

### ISSUE THREAD

**Issue by Erquint** [roskomsvoboda/censortracker#776]
Использую Censor Tracker исключительно в информационных целях — весь функционал, кроме смены иконки, отключён в настройках.  

На запуске браузера наблюдается такая картина:  
![image](https://github.com/user-attachments/assets/2b5eddbe-2488-4fa2-84c2-e303cfc0bdc6)
Сверху коренной процесс браузера непосредственно запущенный из графического shell'а.  
Он и его соучастник (по всей видимости extension host Censor Tracker'а) стримят и обрабатывают данные с диска около 300 МБ/сек, несколько минут взрывая оперативную память.  
Счётчик памяти в правом-нижнем углу иллюстрационного скриншота учитывает только физуху — файл подкачки набивается также доверху и выходит в сумме 32 ГБ. При этом сжатие памяти надрывается на 1.25× — то бишь в несжатой сумме расширение приводит к потреблению 40 ГБ памяти!  
О бессмысленной нагрузке на процессор даже упоминать смысла нет при такой картине.  
Через несколько минут прожарки буйство утихает до вменяемых показателей.  

Если я совсем всё на системе не закрою, чтобы дать этому взрыву место — браузер умрёт в попытке запуститься.  
Вкладок у меня в перзистентной сессии много, но имейте рассудок — никто вас не просит все из них немедленно обработать при "ленивой" загрузке браузером на запуске. Они по существу инертны.  
Если отключить ваше одно расширение и оставить остальные 20 включёнными — всё гладко. Дурдом только с ним.  
Проблема наблюдается давно, но уже невозможно это терпеть и от использования придётся отказаться.  

40 гигов дляради чтобыдабы не сделать ровным счётом ничего! У меня криптомайнеры столько не жрали…

**Comment by rellikmil** [comment]
Да, в Firefox у меня тоже самое иногда бывает, после обновления пропало, потом после другого обновления опять также было.

**Comment by WarAn22** [comment]
К сожалению, подтверждаю: при старте сжирает до 11 гигов (если они есть), потом быстро отпускает:

![Image](https://github.com/user-attachments/assets/109da6df-2c7c-4336-b50e-a4f10b445b3e)

А если их нет, то Вивальди крашится.

Да, замечено: если вкладок много (а у меня их штук 500 на компе), то жрет память, как не в себя, но если их всего 58 (на моем ноуте), то жрет-то немного - 0.4-0.6 гига, но все равно это длится долго, пару-тройку десятков секунд, и все это время браузер недоступен.
Отчего можно подтвердить мысль @Erquint о том, что плагин сканирует все НЕоткрытые вкладки, что безусловно, является совершенно лишним действием.

Пожалуйста, поправьте это поведение

**Comment by Serega007RU** [comment]
Соглашусь, при небольшой активности на вкладках в диспетчере задач браузера это расширение находится в топе по прожорливости процессора

![Image](https://github.com/user-attachments/assets/a4c1a9e7-1fa2-4211-8136-5c245d3e51a5)

![Image](https://github.com/user-attachments/assets/9688eb6d-c7a2-4eec-80bf-0c810eab04bb)
